src/server/controllers/questions.js

```diff
@@ -69,6 +75,40 @@ const optionsRead = {
   ],
 };
 
+async function getVotes(req, res, next) {
+  if (req.get('Content-Type') === 'text/csv') {
+    const { resource } = req.locals;
+
+    try {
+      const question = await Question.findOne({
+        rejectOnEmpty: true,
+        where: { festivalId: resource.id },
```

Comment:
I think this querying is wrong for the question page - isn't the resource be the question here? I think maybe you want `chainId: resource.chainId`?

Reply:
Actually the whole query is unnecessary, resource is already the question I'm fetching from here.